scanner.py

```diff
@@ -329,6 +335,15 @@ def __init__(self, config: ConfigWrapper):
             desc=self.cmd_Z_OFFSET_APPLY_PROBE_help,
         )
 
+    def is_valid_model_loaded(self, gcmd: GCodeCommand):
```

Comment:
```suggestion
    def validate_model_loaded(self, gcmd: GCodeCommand):
```
I'd change the name `is_` makes it sound like a boolean that you have to react to, this is more a statement.
"Is a valid model loaded? Yes - ..., No - ..."
versus
"Validate that a model is loaded" - more of a command, doesn't require a response.

Reply:
Yeah makes sense, it originally was a bool :D